### PR TITLE
docs: add v2020.1.1 release notes

### PR DESCRIPTION
**Pull request by blocktrron**

### CODE REVIEW

File: docs/releases/v2020.1.1.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Gluon 2020.1.1
+==============
+
+This is the first service release for the Gluon 2020.1.x line, fixing bugs reported by the community.
```
Comment (mweinelt) [inline docs/releases/v2020.1.1.rst]:
```suggestion
This is the first service release for the Gluon 2020.1.x line, fixing regressions reported by the community.
```

File: docs/releases/v2020.1.1.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- Fixed non-working LEDs on TP-Link Archer C5 v1 and Archer C7 v2 after an upgrade to Gluon 2020.1.
+
+- Fixed an issue which leads to AVM FRITZ!WLAN Repeater 450E devices being stuck in failsafe mode
+  ater an upgrade to Gluon 2020.1.
+
+Other changes
+-------------
+
+- Linux kernel has been updated to 4.14.171
```
Comment (mweinelt) [inline docs/releases/v2020.1.1.rst]: Missing known issues.

Comment (blocktrron) [reply docs/releases/v2020.1.1.rst]: Well, we missed these with the v2020.1 also. But I'll add them back here

Comment (mweinelt) [reply docs/releases/v2020.1.1.rst]: Also add the regression we know about now.

Comment (blocktrron) [reply docs/releases/v2020.1.1.rst]: I've included the ER-X one, are there more?

Comment (mweinelt) [reply docs/releases/v2020.1.1.rst]: That depends on if we want to add the regressions fixed in this release.

Comment (rotanid) [reply docs/releases/v2020.1.1.rst]: if we missed them in v2020.1 we should add them to the old release notes now too now that we noticed it

Comment (mweinelt) [reply docs/releases/v2020.1.1.rst]: Fixed in c219d3c1dc6e5612273daf3f87437df526c400a3 and 9e4eb182908320cb94974663c7d201ce792eea78.

**Pull request merged**